src/Form.elm

```diff
@@ -1574,7 +1574,9 @@ type alias Model =
     Dict String FormState
 
 
-{-| Initialize the [`Form.Model`](Form#Model).
+{-| Initialize an empty form model. This managed state for [`Form.Model`](Form#Model).
+
+Note: Most applications should only need a single `Form.Model` in their application.
```

Comment:
Might be good to clarify this a bit more. I think you're trying to make it clear to people that they don't need a different one for each page, right?

Maybe it's worth elaborating on the tradeoffs around this?

For example, it is okay to re-use the same `Form.Model` across different pages, and this will result in maintaining page state between page views.

But it's worth considering whether you would want to preserve some form state when you change routes, or whether you want to clear it out on page change, etc. Any thoughts on that?

Reply:
I see. That sounds like a tricky explanation. I'll go ahead and remove the note for now. Maybe we elaborate more on the different choices you could make depending on the situation.